lxd/network/driver_ovn.go

```diff
@@ -5579,8 +5584,9 @@ func (n *ovn) PeerUpdate(peerName string, req api.NetworkPeerPut) error {
 	}
 
 	newPeer := api.NetworkPeer{
-		Name:           curPeer.Name,
-		NetworkPeerPut: req,
+		Name:        curPeer.Name,
```

Comment:
As req is NetworkPeerPut we should be using api.NetworkPeer.SetWritable() to avoid duplication.